core/tests/webdriverio_utils/ExplorationEditorTranslationTab.js

```diff
@@ -199,6 +199,7 @@ var ExplorationEditorTranslationTab = function() {
 
   this.saveAudioRecord = async function() {
     await action.click('Save uploaded audio button', saveUploadedAudioButton);
+    await action.waitForAutosave();
     await waitFor.pageToFullyLoad();
```

Comment:
Looks like we were using `waitFor.pageToFullyLoad` instead of `action.waitForAutosave()`. You can remove `action.pageToFullyLoad()` if it's no more needed! (Optional)